### PR TITLE
Query parameter handling in external hierarchies

### LINKED ISSUE #9: NoMethodError: undefined method `stringify' for #<My::Error::UserNotFound:0x…>
When `My::Error` does not descend from `KineticCafe::Error`, it cannot be created with a `query` parameter because the private `stringify` method does not exist in `KineticCafe::ErrorModule`, but only in `KineticCafe::Error`.

### PR DESCRIPTION
**Pull request by halostatue**
Fixes #9

### CODE REVIEW

**Comment by halostatue** [comment]
@jsutlovic

File: lib/kinetic_cafe/error.rb [hunk 1]
```diff
@@ -25,7 +25,7 @@ module KineticCafe # :nodoc:
   # rescue clause and handled there, as is shown in the included
   # KineticCafe::ErrorHandler controller concern for Rails.
   class Error < ::StandardError
-    VERSION = '1.8.1' # :nodoc:
+    VERSION = '1.9' # :nodoc:
```
Comment (iirving) [inline lib/kinetic_cafe/error.rb:28]: starting with VERSION = '1.9'.freeze, is it time to "Freeze all the strings"?   Thinking about it, I don't see this being mutated but might there be performance benefits?  I guess one test would be to add the magic comment (# frozen_string_literal: true) and see how (and if) it run under ruby 2.3rc.

Comment (halostatue) [inline lib/kinetic_cafe/error.rb:28]: There’s only performance benefits if people are working with KineticCafe::Error::VERSION. Other strings may be worthwhile freezing, but I’m not sure that there’s a lot of them to freeze here.

**Comment by iirving** [comment]
:+1:

**Comment by jsutlovic** [comment]
:+1:

**Pull request merged**